### PR TITLE
test(streaming): optimized encoder redux

### PR DESCRIPTION
**Pull request by blackxored**
Awaits up to 10sec for Twitch responses for available games.

### CODE REVIEW

File: test/streaming/live-dialog.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+   * "Use optimized encoder settings for Overwatch".
+   * Unfortunately, something seems to be messing up webdriver and it can't find
+   * 'label=Use optimized encoder settings for Overwatch'
+   * so we workaround by adding a CSS class for when it's using the default profile
+   */
+  const owSelector = '.multiselect__element=Overwatch';
+  await app.client.setValue('.multiselect__input', 'Overwa');
+  await app.client.waitForExist(owSelector, 10000);
+  await app.client.click(owSelector);
+  await sleep(1000);
```
Comment (holiber) [inline test/streaming/live-dialog.ts]: that may be flaky, use waitForVisible/waitForExist

File: test/streaming/live-dialog.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    await focusMain(t);
+    await app.client.waitForExist('button=End Stream');
+    await app.client.click('.top-nav .icon-settings');
+
+    await focusChild(t);
+    await app.client.click('li=Output');
+
+    t.is('Advanced', await app.client.getValue('[data-name=Mode] input'));
+    t.is('Software (x264)', await app.client.getValue('[data-name=Encoder] input'));
+    t.is('2500', await getFormInput(t, 'Bitrate'));
```
Comment (holiber) [inline test/streaming/live-dialog.ts]: `FormMonkey` provides a unified way to fill out forms. Also, it prevents some issues with flaky tests

File: test/streaming/live-dialog.ts (hunk 1 above)
Comment (blackxored) [reply test/streaming/live-dialog.ts]: There doesn't seem to be a selector I can tap in (we're looking for input's value as our wait condition) so I did something similar to what FormMonkey seems to do with dropdowns.

File: test/streaming/live-dialog.ts (hunk 2 above)
Comment (blackxored) [reply test/streaming/live-dialog.ts]: IMHO using FormMonkey's `read` is more complex for this test and scenario, even if it works.

**Pull request merged**